sql/4_2_2-to-5_0_0_upgrade.sql

```diff
@@ -1252,7 +1252,7 @@ CREATE TABLE `form_eye_mag_prefs` (
   `selection` varchar(255) DEFAULT NULL,
   `ZONE_ORDER` int(11) DEFAULT NULL,
   `GOVALUE` varchar(10) DEFAULT '0',
-  `ordering` tinyint(4) DEFAULT NULL,
+  `ordering` smallint(6) DEFAULT NULL,
```

Comment:
revert this. no need to worry about prior upgrade scripts (and it could lead to odd bugs).

Reply:
Done